metpetdb_api/api/samples/v1/views.py

```diff
@@ -422,3 +428,70 @@ def get(self, request, format=None):
             .distinct()
         )
         return Response({'sample_owner_names': sample_owner_names})
+
+'''
+class SampleCSVRenderer (r.CSVRenderer):
+    header = ['Sample', 'Rock_Type', 'Comment', 'Latitude', 'Longitude', 'Location_Error', 'Region', 'Country', 'Collector', 'Date_of_Collection', 'Present_Sample_Location', 'Reference', 'Metamorphic_Grade', 'Minerals', 'Subsamples', 'Chemical_Analyses']
```

Comment:
I'd include this as part of a constructor as explained above.
`def __init__(self):
....`